nncf/torch/nncf_network.py

```diff
@@ -1132,3 +1270,26 @@ def compression_module_type_to_attr_name(compression_module_type: ExtraCompressi
     if compression_module_type == ExtraCompressionModuleType.EXTERNAL_OP:
         return EXTERNAL_OP_STORAGE_NAME
     raise nncf.ValidationError("Unknown extra module type")
+
+
+def create_pt_insertion_command(
```

Comment:
What do you think about moving this function to `commands.py`?

Reply:
Agree, done

Reply:
Moved to `command_creation.py`